compiler-rt/lib/msan/msan.h

```diff
@@ -321,6 +321,9 @@ const int STACK_TRACE_TAG_VPTR = STACK_TRACE_TAG_FIELDS + 1;
     stack.Unwind(pc, bp, nullptr, common_flags()->fast_unwind_on_fatal); \
   }
 
+#define GET_FATAL_STACK_TRACE \
```

Comment:
Add a comment that this is to complement `GET_MALLOC_STACK_TRACE` when not tracking origins.

Reply:
I think it's redundant here, we have similar macro in other sanitizers, and it's just shortcut for GET_FATAL_STACK_TRACE_PC_BP.

Maybe these way as in new snapshot?